build.zig.zon

```diff
@@ -22,8 +22,8 @@
             .hash = "1220e75685f00c242fbf6c1c60e98dd1b24ba99e38277970a10636e44ed08cf2af8a",
         },
         .treesitter = .{
-            .url = "git+https://github.com/tree-sitter/tree-sitter?ref=v0.25.3#dcdd5bc372dae42a14deafedba826d91f02a1ab0",
-            .hash = "tree_sitter-0.26.0-Tw2sR3K4CwA_GbHqW5STjLdERuYa_LBew1nhD3WG-vSw",
+            .url = "git+https://github.com/tree-sitter/tree-sitter?ref=v0.25.4#31b9717ca3d674d6166f8e4aa17e7927b027c1e1",
+            .hash = "tree_sitter-0.26.0-Tw2sR3C5CwBpi7K_Vn9EkVcnPXZegnJ6H0F8WANJhnKB",
```

Comment:
@bfredl that hash prefix doesn't look right?

Reply:
it is from https://github.com/tree-sitter/tree-sitter/blob/master/build.zig.zon#L4

Ideally tree-sitter would set the correct value in point-releases, but this doesn't cause any problems per se.

Reply:
But patch releases come from https://github.com/tree-sitter/tree-sitter/blob/release-0.25/build.zig.zon

Reply:
yeah, there was a bug in the script https://github.com/neovim/neovim/pull/33987

and https://github.com/tree-sitter/tree-sitter/commit/185076211821953f1e765cec4f8b24dbb16f47f6 would need a backport into release-0.25 as well

Reply:
Doesn't apply cleanly, unfortunately, and I don't understand the format enough to resolve the conflicts. Mind making a manual backport PR?